diff --git a/input.conf b/input.conf
index 608e983..3138790 100644
--- a/input.conf
+++ b/input.conf
@@ -15,7 +15,13 @@ WHEEL_DOWN 	add volume -5
 # ──────────────────────────────────────────────────────────────────────
 MBTN_LEFT cycle pause
 MBTN_RIGHT ignore
-MBTN_LEFT_DBL ignore
+MBTN_LEFT_DBL cycle fullscreen
+
+
+# Volume control
+UP add volume 5; show-text "Volume ${volume}%"
+DOWN add volume -5; show-text "Volume ${volume}%"
+
 
 # ──────────────────────────────────────────────────────────────────────
 # Script keybindings
@@ -48,4 +54,4 @@ s script-binding screenshotfolder/screenshot_done
 ctrl+f script-binding selectformat/menu
 
 g script-binding sponsorblock/set_segment
-ctrl+g script-binding sponsorblock/submit_segment
\ No newline at end of file
+ctrl+g script-binding sponsorblock/submit_segment
diff --git a/mpv.conf b/mpv.conf
index a004ae8..e0ab26f 100644
--- a/mpv.conf
+++ b/mpv.conf
@@ -7,7 +7,9 @@ osd-duration=2000														# how long osd messages show, in ms
 osd-font="Segoe UI Semibold" 											# UI font, must be available on your system, 'Noto Sans' is used often on Linux
 osd-font-size=30
 osd-outline-size=2
-
+osd-on-seek=msg
+osd-status-msg=${time-pos} / ${duration}
+osd-align-x=right
 # ──────────────────────────────────────────────────────────────────────
 # Watch Later - https://mpv.io/manual/master/#watch-later
 # ──────────────────────────────────────────────────────────────────────
@@ -100,4 +102,4 @@ profile-cond=height > 1080 and not (audio_codec and (container_fps == nil or con
 [audio]
 sub-font="A-OTF Shin Go Pro M"
 # script-opts-append=modernx-persistent_progress=yes
-profile-cond=audio_codec and (container_fps == nil or container_fps == 1)
\ No newline at end of file
+profile-cond=audio_codec and (container_fps == nil or container_fps == 1)
